sample/src/main/java/dev/marcellogalhardo/retained/sample/SampleView.kt

```diff
@@ -0,0 +1,21 @@
+package dev.marcellogalhardo.retained.sample
+
+import android.content.Context
+import android.util.AttributeSet
+import android.util.Log
+import android.view.View
+import dev.marcellogalhardo.retained.view.retain
+
+class SampleView(context: Context, attrs: AttributeSet) : View(context, attrs) {
```

Comment:
Oh, shouldn't you have a `@JvmOverloads` here? 🤔

Reply:
I just went with the minimum needed in order to make it work to minimize pollution in the example. But I'm actually also not a fan of `@JvmOverloads` on views since [they may cause issues in some occasions](https://zsmb.co/jvmoverloads-for-android-views/).